src/user/dashboard/portfolio/portfolio.js

```diff
@@ -1,40 +1,55 @@
 import React, { Component } from "react";
 import "./portfolio.scss";
+import { data_1, data_2, data_3 } from '../../../jsonData/portfolio-feed'
+import { Typography, ListItemText, ListItem, List } from '@material-ui/core';
+
+
 
 class Portfolio extends Component {
   render() {
+    let portfolio_1 = data_1.map((item, i) => {
+      return (
+        <ListItem className="item" key={i}>
+          <ListItemText className="list-text"
+          primary={<Typography style={{"fontWeight": "bold", "fontSize": "36px"}}>{item.contentValue}</Typography>}
```

Comment:
@rak-shit Please write it as a CSS class and use that class here for primary and secondary.
This will improve our codebase and minimize the tendency of error.
Hope you will do the suggested changes!!